src/arrayMethodSort.js

```diff
@@ -4,8 +4,27 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function(callback) {
+    const compareFunction = callback
+    || ((a, b) => String(a).localeCompare(String(b)));
```

Comment:
Instead of  using || you can give this function as the default value for `callback` argument